config.json

```
@@ -2464,6 +2464,26 @@
           "multiple-clause-matching"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "affine-cipher",
+        "name": "Affine Cipher",
+        "uuid": "18d12bac-ead0-456c-bcef-ccb2fd06fe72",
+        "prerequisites": [
+          "pattern-matching",
+          "atoms",
+          "pipe-operator",
+          "enum",
+          "integers",
+          "if",
```

Let's force people to know all conditional expressions and choose the appropriate one on their own (usually those can be interchangeable anyway)
```suggestion
          "if",
          "cond",
          "case",
```